server/src/main/java/com/cloud/storage/StorageManagerImpl.java

```diff
@@ -2688,7 +2688,8 @@ private void handleRemoveChildStoragePoolFromDatastoreCluster(Set<String> childD
 
     }
 
-    private void updateStoragePoolHostVOAndBytes(StoragePool pool, long hostId, ModifyStoragePoolAnswer mspAnswer) {
+    @Override
+    public void updateStoragePoolHostVOAndBytes(StoragePool pool, long hostId, ModifyStoragePoolAnswer mspAnswer) {
```

Comment:
@weizhouapache, is it possible to add the condition if the pool is StorPool to not set the capacity/used bytes? We have a custom ModifyStorageCommand and this one returns wrong values for us. The correct ones will be applied when the storage stats are collected and meanwhile this could cause issues for the StorPool users

Reply:
thank you @weizhouapache, I hope so too. I'll test it in a bit